__tests__/suites/paypal-button-click-pom.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai";
+import { CheckoutPage } from "../pages/checkoutPage";
+import { HomePage } from "../pages/homePage";
+
+describe("Testing Paypal buttons", () => {
+
+    beforeEach('Open StackDemo', () => {
+        browser.url("https://developer.paypal.com/demo/checkout/#/pattern/client");
+    });
+
+    afterEach('clear sessionstorage', () => {
+        browser.execute(() => sessionStorage.clear())
+    });
+
+    it("should start payment flow when clicking on paypal button", () => {
+        const homePage = new HomePage();
+        homePage.switchToPaypalFrame();
+        homePage.clickPaypalButton();
+        homePage.switchToParentFrame();
+
+        const windows: string[] = browser.getWindowHandles();
+        expect(windows.length).to.equal(2);
+        browser.switchToWindow(windows[1]);
```

Comment:
this could be in a common reusable ts function. let's remove this from this file.

consider when scripters write tests, they have all functions ready - either from pom OR from a reusable file